DangerFile

```diff
@@ -2,9 +2,33 @@ junit.parse "test_output/report.junit"
 junit.show_skipped_tests = true
 junit.report
 
+# Warn when there is a big PR
+warn("Big PR, try to keep changes smaller if you can") if git.lines_of_code > 500
+
+# Stop skipping some manual testing
+warn("Needs testing on a Phone if change is non-trivial") if git.lines_of_code > 50 && !github.pr_title.include?("📱")
```

Comment:
What is this one about?

Reply:
It's just a reminder to test on device if a change is more than 50 loc.  I can up the loc or remove it, I just thought it's a good reminder (especially for for external PR's).